ui/app/components/actions-flyout.hbs

```diff
@@ -0,0 +1,36 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
```

Comment:
contextualActions is the most notable feature of this component; it currently exists if the Ember router detects you're within a job or task page that has actions, but could be expanded in the future.